services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -147,7 +147,7 @@ const AdminUnlockButton = ({
     <Td>
       <Button
         variant="link"
-        disabled={report.locked === false}
```

Comment:
you are inverting the locked value from previous. is that intentional?

Reply:
oops, good catch!